aircan/dags/api_ckan_import_to_bq.py

```diff
@@ -48,11 +48,14 @@
 
 def task_import_resource_to_bq(**context):
     logging.info('Invoking import resource to bigquery')
-    gc_file_url = context['params'].get('resource', {}).get('path')
+    logging.info("resource: {}".format(context['params'].get('resource', {})))
+
+    gc_file_url = context['params'].get('big_query', {}).get('gcs_uri')
```

Comment:
i don't really understand where `gcs_uri` is coming from elsewhere in the code ...

Reply:
We are going to get it from `aircan-connector` - https://github.com/datopian/ckanext-aircan/pull/24